hypothesis-python/tests/py3/test_lookup.py

```diff
@@ -367,6 +367,15 @@ def test_required_args(target, args, kwargs):
               **{k: st.just(v) for k, v in kwargs.items()}).example()
 
 
+class AnnotatedNamedTuple(typing.NamedTuple):
+    a: str
+    i: int
+
+
+def test_required_args_for_namedtuple():
+    print(st.builds(AnnotatedNamedTuple).example())
```

Comment:
No need for a `print()` here.  I'd also add a comment explaining that this test exists because `AnnotatedNamedTuple` does not have an `__init__` method.

Reply:
ahh. forgot to remove it after debugging :)